example/README.md

```diff
@@ -82,7 +82,9 @@ There are several examples showing Dream with Reason syntax.
 - [**`z-heroku`**](z-heroku#files) &nbsp;&mdash;&nbsp; to
   [Heroku](https://www.heroku.com).
 - [**`z-docker`**](z-docker#files) &nbsp;&mdash;&nbsp; on a server, using
-  Docker.
+  Docker with Esy as the package manager.
+- [**`z-docker-opam`**](z-docker#files) &nbsp;&mdash;&nbsp; on a server, using
```

Comment:
The link shold point to `z-docker-opam#files`.

Reply:
Done